### PR TITLE
Kraken: enable disruptions on departures and arrivals

### PR DESCRIPTION
**Pull request by kinnou02**

### CODE REVIEW

File: source/kraken/worker.cpp [hunk 1]
```diff
@@ -230,32 +230,33 @@ pbnavitia::Response Worker::next_stop_times(const pbnavitia::NextStopTimeRequest
...
+    bt::ptime current_datetime = bt::from_time_t(request._current_datetime());
     try {
         switch(api) {
         case pbnavitia::NEXT_DEPARTURES:
             return timetables::next_departures(request.departure_filter(),
                     forbidden_uri, from_datetime,
                     request.duration(), request.nb_stoptimes(), request.depth(),
-                    type::AccessibiliteParams(), *data, false, request.count(),
-                    request.start_page(), request.show_codes());
+                    type::AccessibiliteParams(), *data, true, request.count(),
```
Comment (antoine-de) [inline source/kraken/worker.cpp:240]: don't we want to have an API parameter to display or not the disruptions for those query ?

Not sure it's always relevant.

Comment (kinnou02) [inline source/kraken/worker.cpp:240]: @stifoon don't think it's needed currently

**Comment by l-vincent-l** [comment]
jenkins, test this please

**Comment by l-vincent-l** [comment]
jenkins, test this please

File: source/jormungandr/jormungandr/interfaces/v1/Disruptions.py [hunk 2]
```diff
@@ -137,12 +137,9 @@ def __init__(self):
                                 description="Number of disruptions per page")
         parser_get.add_argument("start_page", type=int, default=0,
                                 description="The current page")
-        parser_get.add_argument("_current_datetime", type=date_time_format, default=datetime.now(), #TODO!! we have to take the local instance time
+        parser_get.add_argument("_current_datetime", type=date_time_format, default=datetime.utcnow(), #TODO!! we have to take the local instance time
                                 description="The datetime we want to publish the disruptions from."
```
Comment (l-vincent-l) [inline source/jormungandr/jormungandr/interfaces/v1/Disruptions.py:141]: Is the comment still needed?

**Pull request merged**